build.gradle

```diff
@@ -22,7 +22,18 @@ task clean() {
 
   doLast {
     exec {
-      executable "./clean.sh"
+      executable "./tools/clean.sh"
+    }
+  }
+}
+
+task ssync() {
+  group = 'Build'
```

Comment:
Perhaps we can rename this group since clean and sync aren't really build tasks?

Reply:
Changed gruop to `Tools` for all of them, I couldn't come up with anything better.

Reply:
That works - matches the folder name too